tee-worker/scripts/launch_local_worker.sh

```diff
@@ -1,80 +1,126 @@
+for ((i = 0; i < ${WORKER_NUM}; i++)); do
+	worker_name="worker${i}"
+	echo ""
+	echo "--------------------setup worker(${worker_name})----------------------------------------"
 
-for ((i = 0; i < workers; i++)); do
 	if ((i > 0)); then
-		option_request_state="--request-state"
+		FSUBCMD_REQ_STATE="--request-state"
 	fi
 
-	worker_name="worker${i}"
-	if [ "${reset}" = 'true' ]; then
-		echo "clear dir: $ROOTDIR/tmp/${worker_name}"
-		rm -rf "$ROOTDIR"/tmp/"${worker_name}"
+	if [ "${CLEANUP}" = 'true' ]; then
+		echo "clear dir: ${ROOTDIR}/tmp/${worker_name}"
+		rm -rf "${ROOTDIR}"/tmp/"${worker_name}"
 	fi
-	mkdir -p "$ROOTDIR"/tmp/"${worker_name}"
+	mkdir -p "${ROOTDIR}"/tmp/"${worker_name}"
 	for Item in 'enclave.signed.so' 'key.txt' 'spid.txt' 'integritee-service' 'integritee-cli'; do
-		cp "${ROOTDIR}/bin/${Item}" "$ROOTDIR"/tmp/"${worker_name}"
+		cp "${ROOTDIR}/bin/${Item}" "${ROOTDIR}"/tmp/"${worker_name}"
 	done
-	echo ""
-	echo "--------------------setup worker(${worker_name})----------------------------------------"
 
 	cd "${ROOTDIR}"/tmp/${worker_name} || exit
 	echo "enter ${ROOTDIR}/tmp/${worker_name}"
 
-	mu_ra_port=$((3443 + i))
-	untrusted_http_port=$((4545 + i))
-	trusted_worker_port=$((2000 + i))
-	untrusted_worker_port=$((3000 + i))
+	mu_ra_port=$((${MU_RA_PORT} + i))
+	untrusted_http_port=$((${UNTRUSTED_HTTP_PORT} + i))
+	trusted_worker_port=$((${TRUSTED_WORKER_PORT} + i))
+	untrusted_worker_port=$((${UNTRUSTED_WORKER_PORT} + i))
 	echo "${worker_name} ports:
-		mu-ra-port: ${mu_ra_port}
-		untrusted-http-port: ${untrusted_http_port}
-		trusted-worker-port: ${trusted_worker_port}
-		untrusted-worker-port: ${untrusted_worker_port}
+	mu-ra-port: ${mu_ra_port}
+	untrusted-http-port: ${untrusted_http_port}
+	trusted-worker-port: ${trusted_worker_port}
+	untrusted-worker-port: ${untrusted_worker_port}
 	"
 
-	launch_command="RUST_LOG=${RUST_LOG} ./integritee-service ${option_clean} \
---mu-ra-external-address ${worker_endpoint} --mu-ra-port ${mu_ra_port} --untrusted-http-port ${untrusted_http_port} --ws-external \
---trusted-external-address wss://${worker_endpoint} --trusted-worker-port ${trusted_worker_port} \
---untrusted-external-address ws://${worker_endpoint} --untrusted-worker-port ${untrusted_worker_port} \
---node-url ${node_url} --node-port ${node_port} \
-run --skip-ra ${option_dev} ${option_request_state}"
+	launch_command="RUST_LOG=${RUST_LOG} ./integritee-service ${F_CLEAN} --ws-external \
+--mu-ra-external-address ${WORKER_ENDPOINT} \
+--mu-ra-port ${mu_ra_port} \
+--node-port ${NODE_PORT} \
+--node-url ${NODE_URL} \
+--trusted-external-address wss://${WORKER_ENDPOINT} \
+--trusted-worker-port ${trusted_worker_port} \
+--untrusted-external-address ws://${WORKER_ENDPOINT} \
+--untrusted-http-port ${untrusted_http_port} \
+--untrusted-worker-port ${untrusted_worker_port} \
+--twitter-official-url ${TWITTER_OFFICIAL_URL} \
+--twitter-litentry-url ${TWITTER_LITENTRY_URL} \
+--twitter-auth-token ${TWITTER_AUTH_TOKEN} \
```

Comment:
Those values should be directly loaded from config files, and should not be passed in as command-line arguments. For example, if we want to add a new config field, we don't need to change the launch script again.